GammaRadiation/GammaRadiationController.py

```diff
@@ -89,20 +92,24 @@
 elif args.admin:
     exp = Experiment("GammaRadiation", admin=True)
 else:
-    exp=Experiment("GammaRadiation", messenger=True)
+    exp = Experiment("GammaRadiation", messenger=True)
+
+peripherals = [camera, buttons, GRpdu]
 
-devices = [camera,
-           stage,
-           actuator,
-           magnet,
-           absorberController,
-           buttons,
-           GRpdu]
+actuators = [stage, actuator, magnet,
+            absorberController]
 
-for device in devices:
+for device in peripherals:
     exp.add_device(device)
+    exp.add_lock([device])
 
-exp.add_lock(devices)
+if args.admin:
```

Comment:
@Chutian-Wang I just want to verify I understand what is happening here. Things are split so that when you run admin you can control them from the browser but otherwise each of the devices inside of the absorbercontroller can't be controlled?

Reply:
Yep that is the plan. Lmk if you think this approach may have problems.